assets/javascripts/discourse/components/minimal-gamification-leaderboard.js

```diff
@@ -0,0 +1,65 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import LoadMore from "discourse/mixins/load-more";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Component.extend(LoadMore, {
```

Comment:
Do you need the `LoadMore` mixin here? If it's only going to load n items, it feels like it's not necessary.

Reply:
We could also use a Glimmer component syntax here, I don't see any blockers.